.github/workflows/main.yml

```diff
@@ -16,6 +16,7 @@ name: build
 
 permissions:
   contents: read
+  id-token: write
```

Comment:
```suggestion
  id-token: write
  packages: write
```

Maybe try this?

Reply:
@imjasonh  Thanks, I tried that but still run into this.

```
2022/04/14 20:51:54 error during command execution:failed to publish images: error publishing ko://github.com/ossf/scorecard/v4: POST https://ghcr.io/v2/naveensrinivasan/scorecard/v4/blobs/uploads/: DENIED: installation not allowed to Create organization package
```